### DISCUSSION TITLE
Everything seems to be configured... I can't start video stream

### DISCUSSION THREAD

**Discussion by DaddyRich1972** [GerryDazoo/Slinger#527] [Q&A]
Hi everyone,

This is a great forum and I have gotten the unified_config.ini and slingbox.py running on my mac. However, I can't get a video stream, both with VLC (I know, not recommended) and O Player lite.

This is what it spits out when I run the program:
`Version : 3.08f Running on macOS-11.5.1-x86_64-i386-64bit pid= 51990 slingbox_server.py
Using config file unified_config.ini
Connection Manager Running on port 8080 with 3 max streams using URL Slingbox.
BOXES [('sb1', '500'), ('sb2', 'Solo'), ('sb3', 'M1')]
Warning TCP buffering might not be sufficent for reliable streaming
Warning TCP buffering might not be sufficent for reliable streaming
Streamer Running:  3 unified_config.ini 500 500 8080 7340032
Streamer Running:  3 unified_config.ini Solo Solo 8080 7340032
Warning TCP buffering might not be sufficent for reliable streaming
Warning TCP buffering might not be sufficent for reliable streaming
Checking for slingbox at ('192.168.4.42', 5201)
Streamer Running:  3 unified_config.ini M1 M1 8080 7340032
Checking for slingbox at ('192.168.117.101', 5001)
Checking for slingbox at ('192.168.117.129', 5301)
starting up on  port 8080 7340032
('192.168.4.42', 5201) OK
500 Using slingbox at  ('192.168.4.42', 5201)
Streamer:  500 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Building page for 500
Reading Custom Remote definition from 500_remote.txt
Error reading remote definition file, using defaults
Building page for Solo
Reading Custom Remote definition from solo_remote.txt
Error reading remote definition file, using defaults
Building page for M1
Reading Custom Remote definition from m1_remote.txt
Error reading remote definition file, using defaults
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
Error connecting to  ('192.168.117.101', 5001)
Error connecting to  ('192.168.117.129', 5301)
Solo No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  192.168.4.33
M1 No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  192.168.4.33
Slingbox Found 192.168.4.42 5201 " My Slingbox " FinderID 6FE11DD2FD30EAB0377FA27AFE59FB42
Slingbox Found 192.168.4.42 5201 " My Slingbox " FinderID 6FE11DD2FD30EAB0377FA27AFE59FB42
M1 Using slingbox at  ('192.168.4.42', 5201)
Solo Using slingbox at  ('192.168.4.42', 5201)
Streamer:  M1 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Streamer:  Solo Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Hacker Alert. Invalid Request from  ('192.168.4.33', 51548)
GET /slingbox HTTP/1.1
Host: 192.168.4.33:8080
Accept: */*
Accept-Language: en_US
User-Agent: VLC/3.0.17.3 LibVLC/3.0.17.3
Range: bytes=0-
`
Every time I try to open the stream this comes up:

`Hacker Alert. Invalid Request from  ('192.168.4.33', 51549)
GET /slingbox HTTP/1.0
Host: 192.168.4.33:8080
User-Agent: VLC/3.0.17.3 LibVLC/3.0.17.3
Icy-MetaData: 1

Hacker Alert. Invalid Request from  ('192.168.4.33', 51550)
GET /slingbox HTTP/1.1
User-Agent: libmpv
Accept: */*
Range: bytes=0-
Connection: close
Host: 192.168.4.33:8080
Icy-MetaData: 1
Hacker Alert. Invalid Request from  ('192.168.4.33', 51549)
GET /slingbox HTTP/1.0
Host: 192.168.4.33:8080
User-Agent: VLC/3.0.17.3 LibVLC/3.0.17.3
Icy-MetaData: 1

Hacker Alert. Invalid Request from  ('192.168.4.33', 51550)
GET /slingbox HTTP/1.1
User-Agent: libmpv
Accept: */*
Range: bytes=0-
Connection: close
Host: 192.168.4.33:8080
Icy-MetaData: 1
`

From what I understand, this is the URL I should be using: 192.168.4.33:8080/slingbox or 192.168.4.33:8080/slingbox/500 - neither one works.

Would appreciate any advice from this helpful community!

**Comment by GerryDazoo** [comment]
You've got issues with you config file you've got 3 slingboxes configure but only one on your network. But assuming you haven't changed your [SERVER] URLbase  then
http://192.168.4.xx:8080/slingbox/500  should work. That URL doesn't show up in your in your logs.  Don't forget the http://

**Reply by DaddyRich1972** [reply]
Thanks Gerry. I only have one Slingbox - should I delete the info in the config file for the others ones? I did use the http:// but no luck...

**Reply by GerryDazoo** [reply]
Yes of course delete the lingboxes your not using or comment them out by prepending a ; before the entries in the [SLINGBOXES] section. 
You need to post your config.ini and your server output. AND stop using the brain-damaged iOS VLC.

**Reply by GerryDazoo** [reply]
Please give this code a try. 
[slingbox_server.zip](https://github.com/GerryDazoo/Slinger/files/10722249/slingbox_server.zip)
